### ISSUE TITLE
Do not abort backup if can't fetch one of the remote target

### ISSUE THREAD

**Issue by Azertooth** [digint/btrbk#468]
As title, I configured a multi remote target backup but sometimes one of those remote nodes is not reachable, there's a way to skip those node's backup and do the rest? Because right now even if one is not reachable the backup fails for all of the remote target

Sample configuration
```
volume
  [...]
  subvolume
    target send-receive <target1>
      ssh_identity [...]
    target send-receive <target2>
      ssh_identity [...]
    target send-receive <target3>
      ssh_identity [...]
```

**Comment by emirror-de** [comment]
I do have a similar problem. My configuration is to keep one daily snapshot and hourly of the current day. When the remote target is not available, the snapshots from the last day are not getting removed so they sum up until the remote target is available again. This is quite annoying when the remote target is more than a few days not available.

**Comment by luxagen** [comment]
I agree. Even if not configurable, the default should be to make target failures a non-fatal error and just report them all at the end. For instance, I do a nightly btrbk run with multiple targets for different subvolumes, and I wouldn't want it to fail the whole job (i.e. the other subvols) just because one had an unreachable target.
